Integrations/Elasticsearch_v2/Elasticsearch_v2.py

```diff
@@ -169,14 +244,20 @@ def fetch_params_check():
     if FETCH_QUERY == '' or FETCH_QUERY is None:
         str_error.append("Query by which to fetch incidents is not configured.")
 
-    if TIME_FORMAT == '' or TIME_FORMAT is None:
+    if (TIME_FORMAT == '' or TIME_FORMAT is None) and TIME_METHOD == 'Simple-Date':
```

Comment:
shouldn't it be an error for all cases?

Reply:
no, in case it is a timestamp the TIME_FORMAT is not necessary because the TIME_FORMAT is predefined (lines 45-46)